### PR TITLE
#162412144 Integrate PostgreSQL database

### PR DESCRIPTION
**Pull request by devskope**
**What does this PR do?**

Sets up a postgresql database.

**Description of Task to be completed?**
- Setup postgreSQL database

**How should this be manually tested?**

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

162412144

Screenshots (if appropriate)

N/A

Questions:

N/A

### CODE REVIEW

File: src/db/dbrc.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import debug from 'debug';
+import db from './db';
+
+const logger = debug('iReporter::initdb:');
+const Models = {
+  recordSchema: `
```
Comment (fob413) [inline src/db/dbrc.js:6]: is this meant to be `recordSchema` or `userSchema`?

File: src/db/dbrc.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+       othernames VARCHAR,
+       username  VARCHAR UNIQUE NOT NULL,
+       email  VARCHAR NOT NULL,
+       phone  VARCHAR,
+       registered TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+       is_admin BOOLEAN DEFAULT 'false'
+      )`,
+
+  userSchema: `
+    CREATE TABLE IF NOT EXISTS records
```
Comment (fob413) [inline src/db/dbrc.js:21]: is this meant to be `userSchema` or `recordSchema`?

File: src/db/db.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { Pool } from 'pg';
+import env from '../config/envConf';
+
+const pool = new Pool({
+  connectionString: env.DATABASE_URL,
```
Comment (fob413) [inline src/db/db.js:5]: I can't seem to locate `env.DATABASE_URL` in your `envConf` file. Please clarify.

**Review by fob413 — CHANGES_REQUESTED** [review]

File: src/db/dbrc.js (hunk 1 above)
Comment (devskope) [reply src/db/dbrc.js:6]: should be `userShema`   fixed in #32

File: src/db/dbrc.js (hunk 2 above)
Comment (devskope) [reply src/db/dbrc.js:21]: should be `recordShema`   fixed in #32

File: src/db/db.js (hunk 3 above)
Comment (devskope) [reply src/db/db.js:5]: I have explicitly added the variable in #32

**Pull request merged**